pycsw/core/config.py

```diff
@@ -175,6 +175,8 @@ def __init__(self, prefix='csw30'):
                 'pycsw:Bands': 'bands',
                 # links: list of dicts with properties: name, description, protocol, url
                 'pycsw:Links': 'links',
+                # contacts: list of dicts with properties: name, description, protocol, url
```

Comment:
Update comment to describe stored contacts objects

Reply:
This has been resolved by latest commit